cpp/src/io/comp/uncomp.cpp

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
+#include "io/utilities/hostdevice_vector.hpp"
 #include "io_uncomp.h"
 #include "unbz2.h"  // bz2 uncompress
 
+#include <cudf/detail/utilities/vector_factories.hpp>
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/span.hpp>
+#include <io/comp/nvcomp_adapter.hpp>
```

Comment:
Local header to be placed before `#include "io/utilities/hostdevice_vector.hpp"` and decorated with double quotes.